Alignment/CommonAlignmentProducer/plugins/PCLTrackerAlProducer.cc

```diff
@@ -507,7 +507,7 @@ ::createGeometries(const edm::EventSetup& setup)
     setup.get<PTrackerParametersRcd>().get( ptp );
 
     theTrackerGeometry = boost::shared_ptr<TrackerGeometry>(
-        trackerBuilder.build(&(*geometricDet), *ptp )
+        trackerBuilder.build(&(*geometricDet), *ptp, tTopo )
```

Comment:
Discussed offline and decided to leave it like this for now - space for improvements for later.